app/uk/gov/hmrc/emcstfe/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.featureswitch.core.config
+
+import play.api.inject.{Binding, Module}
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.emcstfe.featureswitch.core.models.FeatureSwitch
+
+import javax.inject.Singleton
+
+@Singleton
+class FeatureSwitchingModule extends Module with FeatureSwitchRegistry {
+
+  val switches: Seq[FeatureSwitch] = Seq(ChrisStubEnabled)
+
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
+    Seq(
+      bind[FeatureSwitchRegistry].to(this).eagerly()
+    )
+  }
+}
+
+case object ChrisStubEnabled extends FeatureSwitch {
+  override val configName: String = "features.chrisStub"
```

Comment:
You'll need to enable this in the environment app-configs as well

Reply:
PR is was put up a couple of days for QA since its the only place it needs to be enabled after a dissuccsion with Matt, if you really want me too I can put it explicitly for every config

Reply:
No that's fine, just making sure there was a change for it